src/sinks/influxdb/logs.rs

```diff
@@ -44,18 +44,24 @@ pub struct InfluxDbLogsConfig {
     ///
     /// When specified, the measurement name will be `<namespace>.vector`.
     ///
-    /// This field is deprecated, and `measurement` should be used instead.
-    #[configurable(deprecated)]
+    #[configurable(deprecated = "This field is deprecated, and `measurement` should be used instead.")]
+    #[configurable(metadata(docs::examples = "service"))]
     pub namespace: Option<String>,
 
     /// The name of the InfluxDB measurement that will be written to.
+    #[configurable(metadata(docs::examples = "vector-logs"))]
     pub measurement: Option<String>,
 
     /// The endpoint to send data to.
+    ///
+    /// This should be a full HTTP URI, including the scheme, host, and port.
+    #[configurable(metadata(docs::examples = "http://localhost:8086/"))]
```

Comment:
nit (I know it was this way in the existing docs)

```suggestion
    #[configurable(metadata(docs::examples = "http://localhost:8086"))]
```